conftest.py

```diff
@@ -25,10 +25,12 @@
 logger = logging.getLogger(__name__)
 
 
-def pytest_sessionstart():
+@pytest.fixture(scope="session", autouse=True)
+def testdata():
     """ Download sct_testing_data prior to test collection. """
     logger.info("Downloading sct test data")
-    downloader.main(['-d', 'sct_testing_data', '-o', sct_test_path()])
+    if not os.path.isdir(sct_test_path()):
+        downloader.main(['-d', 'sct_testing_data', '-o', sct_test_path()])
```

Comment:
Reminder note: Update this to match new changes from #3152.

Reply:
I did! It's in the OP. "Blocked by #2957."